tests/ToggleButton.test.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import ToggleButton from '../src/ui/ToggleButton';
+
+describe('ToggleButton', () => {
+  test('Button does not show checkmark if isChecked=false', () => {
+    render(<ToggleButton>hello</ToggleButton>);
```

Comment:
As an introduction to the testing, I would suggest reading through the react-testing-library docs, which give a pretty good intro to how to think about this stuff.

Read through the intro and guiding principles here: https://testing-library.com/docs/

And then read through the intro and examples in the frameworks/react section. A lot of info, but good to start getting familiar with this stuff!